sentry-android-core/src/main/java/io/sentry/android/core/SentryAndroidOptions.java

```diff
@@ -133,6 +130,8 @@ public final class SentryAndroidOptions extends SentryOptions {
 
   private boolean enableFramesTracking = true;
 
+  private final @NotNull List<GestureTargetLocator> gestureTargetLocators = new ArrayList<>();
```

Comment:
`m`: Just thinking if it should be part of `SentryOptions` instead of `SentryAndroidOptions`, since `GestureTargetLocator` lives there? Also, if we ever do UI event transactions/breadcrumbs for compose-desktop, it also needs to be there